packages/synchro-charts/src/components/charts/sc-webgl-base-chart/sc-webgl-base-chart.tsx

```diff
@@ -540,15 +540,18 @@ export class ScWebglBaseChart {
    * Provide no `hasDataChanged` to prevent a re-processing of the chart scenes.
    */
   onUpdate = (
-    { start, end }: { start: Date; end: Date },
-
+    {
+      start,
+      end,
+      shouldBlockDateRangeChangedEvent,
+    }: { start: Date; end: Date; shouldBlockDateRangeChangedEvent?: boolean },
     hasDataChanged: boolean = false,
     hasSizeChanged: boolean = false,
     hasAnnotationChanged: boolean = false,
     shouldRerender: boolean = false
   ) => {
     const hasViewPortChanged = this.start.getTime() !== start.getTime() || this.end.getTime() !== end.getTime();
-    if (hasViewPortChanged) {
+    if (hasViewPortChanged && !shouldBlockDateRangeChangedEvent) {
```

Comment:
Could we just check duration here similarly to how you are in `viewportHandler.ts`?

This way we wouldn't need to pass in any additional properties to `onUpdate`. It also aligns better with your test.

Reply:
That was my initial thought too, but it causes issues. There is a duration passed in any time that `onUpdate` is called, so we'd get false positives for the `isInLiveMode` check when we're performing a gesture or changes on charts in the same viewport group.

With this new property we can be explicit about blocking the event from firing in live mode without changing how other viewport updates work.